src/core/lib/experiments/rollouts.yaml

```diff
@@ -96,6 +96,8 @@
   default: false
 - name: registered_method_lookup_in_transport
   default: true
+- name: rfc_max_concurrent_streams
+  default: true
```

Comment:
Given this is a behavior change to a more delicate piece of code, maybe we start with false here? And roll out internally first?

Reply:
I'm thinking true here for coverage in testing (so it hits *all* the OSS tests), and then a slow rollout internally

Reply:
Hm but this'll also hit any OSS folks building directly right? Why not just debug then?

Reply:
only once we make a release... let's hold until the release branch is cut